gensim/test/test_word2vec.py

```diff
@@ -154,7 +154,7 @@ def testVocab(self):
         total_words = sum(len(sentence) for sentence in corpus)
 
         # try vocab building explicitly, using all words
-        model = word2vec.Word2Vec(min_count=1)
+        model = word2vec.Word2Vec(min_count=1, hs=1, negative=0)
```

Comment:
Could we please add a new test for the new defaults that are introduced in this PR? i.e. where model = word2vec.Word2Vec(min_count=1)